logproof/src/linear_relation.rs

```diff
@@ -1120,6 +1120,7 @@ impl LogProof {
      * This modifies bitvec in place.
      *
      */
+    // TODO seems incorrect for log_b == 1; {0, 1} -> 0 and {-1} -> 1
     fn to_2s_complement_single<B, const N: usize>(value: &Zq<N, B>, log_b: u64, bitvec: &mut BitVec)
```

Comment:
Looking at this more closely, it looks like these `log_b` values get computed by
```rust
|b| if *b > 0 { Log2::ceil_log2(b) + 1 } else { 0 }
```
So 1 should never happen. Perhaps I should add a debug assertion that `log_b != 1`?

Reply:
It should never be 1, correct. 0 is an odd case in this function to essentially remove that element from the mixed bounds check.

Reply:
Added an assertion